bgflow/distribution/normal.py

```diff
@@ -120,8 +124,12 @@ def __init__(
 
         super().__init__(dim=mu.shape)
 
-        self.register_buffer("_mu", mu)
-        self.register_buffer("_sigma", sigma.to(mu))
+        if is_learnable:
+            self._mu = torch.nn.Parameter(mu)
+            self._sigma = torch.nn.Parameter(sigma.to(mu))
+        else:
+            self.register_buffer("_mu", mu)
+            self.register_buffer("_sigma", sigma.to(mu))
```

Comment:
this will cause problems - sigma should be parameterized in log space

Reply:
excellent catch